app/models/hourly_rate.rb

```diff
@@ -58,15 +58,17 @@ def self.at_date_for_user_in_project(date, user_id, project = nil, include_defau
     user_id = user_id.id if user_id.is_a?(User)
 
     unless project.nil?
-      rate = find(:first,
-                  conditions: ['user_id = ? and project_id = ? and valid_from <= ?', user_id, project, date],
-                  order: 'valid_from DESC')
+      rate = where(['user_id = ? and project_id = ? and valid_from <= ?', user_id, project, date])
+             .order('valid_from DESC')
+             .first
       if rate.nil?
         project = Project.find(project) unless project.is_a?(Project)
-        rate = find(:first,
-                    conditions: ['user_id = ? and project_id in (?) and valid_from <= ?', user_id, project.ancestors.to_a, date],
-                    include: :project,
-                    order: 'projects.lft DESC, valid_from DESC')
+        rate = where(['user_id = ? and project_id in (?) and valid_from <= ?',
+                      user_id,
+                      project.ancestors.to_a,
+                      date])
+               .includes(:project)
+               .order('projects.lft DESC, valid_from DESC')
```

Comment:
@NobodysNightmare you missed the call to `.first` here. Corrected in 5cc2403100a6c3d53be04e940314556aa50bc9b0